modules/veteran/lib/tasks/veteran_tasks.rake

```diff
@@ -14,52 +14,14 @@ namespace :veteran do
   task load_sample_vso_data: :environment do
     Veteran::Service::Organization.create(poa: '074', name: '074 - AMERICAN LEGION')
     Veteran::Service::Organization.create(poa: '083', name: '083 - DISABLED AMERICAN VETERANS')
-    Veteran::Service::Organization.create(poa: '095', name: '095 - ITALIAN AMERICAN WAR VETERANS OF THE US, INC.')
-    Veteran::Service::Organization.create(poa: '1NY', name: '1NY - SAMANTHA Y WARSHAUER')
     Veteran::Service::Representative.create(
-      poa: '1NY',
```

Comment:
Just curious why all of these are being removed as part of this -- doesn't seem directly related

Reply:
We needed to update this rake task in order to test it on staging and figured while we are here we also remove all of the unnecessary users that we don't use for VSO/VSR testing and would just confuse and or complicate anyone looking at this in the future.